src/main/java/subway/line/LineRequest.java

```diff
@@ -6,9 +6,18 @@
     private Long upStationId;
     private Long downStationId;
     private int distance;
+    private int extraFare;
 
     public LineRequest() {
     }
+    public LineRequest(String name, String color, Long upStationId, Long downStationId, int distance, int extraFare) {
+        this.name = name;
+        this.color = color;
+        this.upStationId = upStationId;
+        this.downStationId = downStationId;
+        this.distance = distance;
+        this.extraFare = extraFare;
+    }
```

Comment:
다른 시그니쳐를 가진 생성자가 있으므로 아래처럼 바꿀수 있어요
```suggestion
    public LineRequest(String name, String color, Long upStationId, Long downStationId, int distance, int extraFare) {
        this(name, color, upStationId, downStationId, distance);
        this.extraFare = extraFare;
    }